main.nf

```diff
@@ -9,99 +9,31 @@
-     --skip_trimming [bool]             Skip read trimming
-     --clip_r1 [int]                    Trim the specified number of bases from the 5' end of read 1 (or single-end reads).
-     --clip_r2 [int]                    Trim the specified number of bases from the 5' end of read 2 (paired-end only).
-     --three_prime_clip_r1 [int]        Trim the specified number of bases from the 3' end of read 1 AFTER adapter/quality trimming
-     --three_prime_clip_r2 [int]        Trim the specified number of bases from the 3' end of read 2 AFTER adapter/quality trimming
-     --rrbs [bool]                      Turn on if dealing with MspI digested material.
-
-    Trimming presets:
-     --pbat [bool]
-     --single_cell [bool]
-     --epignome [bool]
-     --accel [bool]
-     --zymo [bool]
-     --cegx [bool]
-     --em_seq [bool]
-
-    Other options:
-      --outdir [file]                 The output directory where the results will be saved
-      --publish_dir_mode [str]        Mode for publishing results in the output directory. Available: symlink, rellink, link, copy, copyNoFollow, move (Default: copy)
-      --email [email]                 Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      --email_on_fail [email]         Same as --email, except only send mail if the workflow is not successful
-      --max_multiqc_email_size [str]  Threshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
-      -name [str]                     Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic
-
-    AWSBatch options:
-      --awsqueue [str]                  The AWSBatch JobQueue that needs to be set when running on AWSBatch
-      --awsregion [str]                 The AWS Region for your AWS Batch job to run on
-      --awscli [str]                    Path to the AWS CLI tool
-
-    """.stripIndent()
-}
+log.info Headers.nf_core(workflow, params.monochrome_logs)
 
-// Show help message
+////////////////////////////////////////////////////
+/* --               PRINT HELP                 -- */
+////////////////////////////////////////////////////+
+def json_schema = "$projectDir/nextflow_schema.json"
 if (params.help) {
-    helpMessage()
+    def command = "nextflow run nf-core/methylseq --input '*_R{1,2}.fastq.gz' -profile docker"
+    log.info NfcoreSchema.params_help(workflow, params, json_schema, command)
     exit 0
 }
 
-// Validate inputs
+////////////////////////////////////////////////////
+/* --         VALIDATE PARAMETERS              -- */
+////////////////////////////////////////////////////+
+
 assert params.aligner == 'bwameth' || params.aligner == 'bismark' || params.aligner == 'bismark_hisat' : "Invalid aligner option: ${params.aligner}. Valid options: 'bismark', 'bwameth', 'bismark_hisat'"
```

Comment:
I think that we can delete this now as it should be handled by the schema (assuming that we set up an `enum` there for this param)